writer.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+const _ = require('lodash')
+const prettyOutput = require('prettyoutput')
+
+const internals = {}
+
+/**
+ * Log with pretty output formater
+ * @param {LogWriter} output
+ */
+exports.pretty = output => {
+    const log = _.assign({}, output.meta)
+    output.meta.time = output.meta.time.toISOString()
+    log.data = output.data
+    const prefix = `${output.message}\n`
+    const result = `${prefix}${prettyOutput(log, { maxDepth: 6 }, 2)}`
+    process.stdout.write(result)
+    process.stdout.write('\n')
+}
+
+/**
+ * Log in json
+ * @param {LogWriter} output
+ */
+exports.json = output => {
+    const log = _.assign({}, output.meta)
+    log.time = log.time.toISOString()
+    log.data = output.data
+    log.message = output.message
+
+    const backup = Error.prototype.toJSON
+    Error.prototype.toJSON = internals.errorToJson
+    const result = JSON.stringify(log)
```

Comment:
We should prevent JSON.stringify errors, maybe we should use https://github.com/mcollina/fast-json-parse to handle it nicely with node 6 and 7, or if we just want node 8 support  add a try catch

Reply:
Ok added a try catch and removed support for node <8